cost-analyzer/templates/cost-analyzer-deployment-template.yaml

```diff
@@ -162,6 +178,22 @@ spec:
               mountPath: /var/configs
           securityContext:
             runAsUser: 0
+{{ end }}
+{{- if .Values.supportNFS }}
```

Comment:
Do you feel strongly about opening up another init container to do this? I'd just do this in a few lines of bash on the same init container.